src/validation/rules/optional/ProhibitDeprecatedFieldsRule.d.ts

```diff
@@ -0,0 +1,15 @@
+import { ASTVisitor } from '../../../language/visitor';
+import { ValidationContext } from '../../ValidationContext';
+
+/**
+ * Prohibit deprecated fields
+ *
+ * A GraphQL document is only valid if all selected fields and all used enum
+ * values have not been deprecated.
```

Comment:
For comment/docs/strings you can use more than 80 columns especially if it's just a few words.